org.opentosca.container.core.impl/src/org/opentosca/container/core/impl/plan/PlanInvocationEngine.java

```diff
@@ -1,326 +1,49 @@
 package org.opentosca.container.core.impl.plan;
```

Comment:
Wouldn't it be better to remove the whole when it only does this much? :)

Reply:
In principle yes. The logic for creating the OSGi event could be moved to the calling classes (API, situations). However, we need the possibility to create a correlation ID somewhere in the core. So we could move this method somewhere else, but the question is where? :-)

Reply:
Considering that OSGI is no longer used in the winery-model branch, I don't think it's useful to duplicate the work for that